### DISCUSSION TITLE
No keyboard input

### DISCUSSION THREAD

**Discussion by samgaw58** [Ylianst/MeshCentral#4095] [General]
I have a Mesh Central server running version 1.0.11, one of the client PC's running Win10 also has an old WinXP VM running on VirtualBox. The Win10 PC works as expected without problems, however there is no keyboard input into this VM. There is mouse input into the VM. The PC user tells me that when he used TeamViewer the VM worked without problem.
So my question is, How do I get the keyboard to work on the VM when connected via MeshCentral?

**Comment by krayon007** [comment]
Does it work if you set the remote keyboard mapping checkbox? I routinely use VMWare VMs and can verify it works with that.

**Reply by samgaw58** [reply]
Sadly no, it makes no difference to the keyboard input into the vm.

**Reply by samgaw58** [reply]
To get it to work for now, I turned on Remote Desktop on the VM and setup an RDP session. Everything works fine this way. Although would be so much better if I was able to pass through the keyboard input into virtualbox

**Comment by DMoRiaM** [comment]
I've got the same problem here. My solution, in some cases, was to 
install mesh in the virtual machine itself.

=D'

Em 16/06/2022 12:44, samgaw58 escreveu:
>
> To get it to work for now, I turned on Remote Desktop on the VM and 
> setup an RDP session. Everything works fine this way. Although would 
> be so much better if I was able to pass through the keyboard input 
> into virtualbox
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/Ylianst/MeshCentral/discussions/4095#discussioncomment-2965205>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AGDWLMABTK4LB2M54DDPGXTVPNDVLANCNFSM5YHMSYCQ>.
> You are receiving this because you are subscribed to this 
> thread.Message ID: 
> ***@***.***>
>
-- 
Assinatura de E-mail
DMoRiaM
Nerd, trekker e curioso em tempo integral
Logo <https://www.dmoriam.com.br> 	
José Marcelo Amorim
www.dmoriam.com.br <http://www.dmoriam.com.br>
Para assuntos de trabalho, clique no último ícone!
Facebook icon <https://www.facebook.com/DMoRiaM> LinkedIn icon 
<https://www.linkedin.com/in/dmoriam> Twitter icon 
<https://twitter.com/dmoriam> Youtube icon 
<https://www.youtube.com/user/DMoRiaM> Instagram icon 
<https://www.instagram.com/dmoriam> Pinterest icon 
<https://br.pinterest.com/dmoriam/> Pinterest icon 
<https://www.tepfi.com.br>
